addon/ng2/blueprints/component/files/__path__/__name__.component.ts

```diff
@@ -0,0 +1,16 @@
+import {Component} from 'angular2/core';
+
+@Component({
+  moduleId: __moduleName,
+  selector: '<%= dasherizedModuleName %>',
+  templateUrl: '<%= dasherizedModuleName %>.component.html',
+  styleUrls: ['<%= dasherizedModuleName %>.component.<%= styleExt %>']
+})
+export class <%= classifiedModuleName %>Component {
```

Comment:
It's a good point. We dont want to add things people will remove.

without `ngOnInit` I have noticed many people end up putting that startup code in the ctor. That makes it awful to test. That's why I like it there form a template standpoint. But if you have no startup code, you would need to remove it. I almost always use it. 

That said, I'd be OK removing ngOnInit and the interface and the import if everyone here wants it that way.